gateway/1.4/modules/ROOT/pages/flex-conn-manage-jenkins.adoc

```diff
@@ -470,21 +406,6 @@ post {
 
 ----
 
-* For Kubernetes Ingress Controller Flex Gateway:
```

Comment:
Since there is only one bullet, can you remove it and change the first line of text? Something like:

For Flex Gateways installed in Docker containers, use the following code blocks to stop your Flex Gateway and output the registration file: